cli-toolchain/usage.md

```diff
@@ -83,6 +84,20 @@ The aws secret access key if `useProfile` is set to false.
 - `region`: 
 The aws region if `useProfile` is set to false.
 
+##### `--app`
+Installs, initializes and provisions resources for a sample amplify application from the provided github repository url. This option must be executed in an empty directory. The sample repository must have an amplify folder, including the following:
```

Comment:
Installs, initializes and provisions resources for a sample amplify application from the provided github repository url. -->
Installs, initializes, and provisions resources for a sample amplify application from the provided GitHub repository URL.

Reply:
Made this change